TESTS/integration/COMMON/alice.txt

```diff
@@ -0,0 +1,3736 @@
+Project Gutenberg's Alice's Adventures in Wonderland, by Lewis Carroll
+
+This eBook is for the use of anyone anywhere at no cost and with
+almost no restrictions whatsoever.  You may copy it, give it away or
+re-use it under the terms of the Project Gutenberg License included
```

Comment:
We had a discussion previously about this one as I recall. To my understanding, the license is not acceptable.

Reply:
This file got removed already, it just exist in the git history. do you want me clean up the history and force push a commit as well ? @0xc0170

Reply:
I see, I was probably reviewing only the first commit. please remove from history . Will review afterwards